packages/flutter_tools/test/general.shard/commands/build_aar_test.dart

```diff
@@ -107,6 +108,95 @@ void main() {
     });
   });
 
+  group('flag parsing', () {
+    Directory tempDir;
+    MockAndroidBuilder mockAndroidBuilder;
+
+    setUp(() {
+      mockAndroidBuilder = MockAndroidBuilder();
+      tempDir = globals.fs.systemTempDirectory.createTempSync('flutter_tools_build_aar_test.');
+    });
+
+    tearDown(() {
+      tryToDelete(tempDir);
```

Comment:
This isn't ideal, but it's the pattern the rest of this file uses.  I started yak shaving (hence https://github.com/flutter/flutter/pull/56335, https://github.com/flutter/flutter/pull/56331, https://github.com/flutter/flutter/pull/56330, https://github.com/flutter/flutter/pull/56329) but didn't get far enough to do https://github.com/flutter/flutter/issues/43863 or inject a fake process manager into the command (boy are commands are hard to test...)

Anyway it's no worse than it was...